torchio/data/subject.py

```diff
@@ -214,19 +214,46 @@ def apply_inverse_transform(self, **kwargs) -> 'Subject':
 
     def clear_history(self) -> None:
         self.applied_transforms = []
-
-    def check_consistent_attribute(self, attribute: str) -> None:
-        values_dict = {}
+        
+    def check_consistent_attribute(self,
+                               attribute: str,
+                               rtol=1e-6,
+                               atol=1e-6,
```

Comment:
Can we rename `rtol` and `atol` to something more descriptive? E.g., `relative_tolerance` and `absolute_tolerance` or, if that's too long, `rel_tolerance` and `abs_tolerance`.

It would be also nice to have a little docstring explaining what these are, with at least one example.

Reply:
Will add that :)